v2/internal/frontend/dispatcher/dispatcher.go

```diff
@@ -38,6 +38,8 @@ func (d *Dispatcher) ProcessMessage(message string, sender frontend.Frontend) (s
 		return d.processEventMessage(message, sender)
 	case 'C':
 		return d.processCallMessage(message, sender)
+	case 'c':
```

Comment:
maybe a future change, but could "S" become "D" for display (head out the gutter) letting "S" mean "Secure"

I can see you having future issues with support over "s" and "S"